core/templates/dev/head/pages/topics_and_skills_dashboard/topics_and_skills_dashboard.html

```diff
@@ -1,2 +1,227 @@
 {% extends 'pages/base.html' %}
-<script src="{{TEMPLATE_DIR_PREFIX}}/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardBackendApiService.js"></script>
+{% block maintitle %}
+  Topics and Skills Dashboard - Oppia
+{% endblock maintitle %}
+
+{% block navbar_breadcrumb %}
+  <topics-and-skills-dashboard-navbar-breadcrumb>
+  </topics-and-skills-dashboard-navbar-breadcrumb>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <topics-and-skills-dashboard-navbar>
+  </topics-and-skills-dashboard-navbar>
+{% endblock local_top_nav_options %}
+
+
+{% block content %}
+  <background-banner></background-banner>
+  <div class="oppia-dashboard-container" ng-controller="TopicsAndSkillsDashboard" style="position: relative;">
+    <div>
+      <h2 class="oppia-dashboard-title" translate="I18N_DASHBOARD_TOPICS_AND_SKILLS_DASHBOARD"></h2>
+    </div>
+    <div ng-if="topicSummaries.length > 0 || skillSummaries.length > 0">
+      <ul class="oppia-dashboard-tabs">
+        <li ng-if="topicSummaries.length > 0"
+            ng-class="{'oppia-dashboard-tabs-active': activeTab === 'topics'}">
```

Comment:
Here and elsewhere below, see if you can use constants for the tab names.

Reply:
Done.